templates/default/systemd/docker.service-override.erb

```diff
@@ -26,7 +26,7 @@ ExecStartPre=/sbin/sysctl -w net.ipv6.conf.all.forwarding=1
 ExecStart=<%= @docker_daemon_cmd %>
 ExecStartPost=<%= @docker_wait_ready %>
 Restart=always
-MountFlags=private
```

Comment:
Should the default actually be private since the override is private? Excuse my ignorance to systemd here if this is a crazy question.